libp2p/Host.cpp

```diff
@@ -526,25 +520,29 @@ void Host::addNode(NodeID const& _node, NodeIPEndpoint const& _endpoint)
         return;
     }
 
-    if (_endpoint.tcpPort() < 30300 || _endpoint.tcpPort() > 30305)
+    if (_endpoint.tcpPort() < c_minListenPort || _endpoint.tcpPort() > c_maxListenPort)
         cnetdetails << "Non-standard port being recorded: " << _endpoint.tcpPort();
 
     addNodeToNodeTable(Node(_node, _endpoint));
 }
 
 void Host::requirePeer(NodeID const& _n, NodeIPEndpoint const& _endpoint)
```

Comment:
not cool that this silently fails, it should better return bool. 
But I guess this doesn't change anything now, so we can leave it for now

Reply:
Will file an issue to track and address in a future change

Reply:
> 
> 
> Ok to me, but squash some commits.

I started to work on squashing but ran into a lot of conflicts...it's probably easier to just squash and merge, would that be ok?

Reply:
Filed issue: #5456